### PR TITLE
deploy and undeploy support for submariner

### PR DESCRIPTION
**Pull request by dushyantbehl**

### CODE REVIEW

**Comment by dushyantbehl** [comment]
Linked to this https://github.com/netobserv/multi-cloud-research/issues/1

**Comment by eranra** [comment]
@dushyantbehl, thanks for working on this :-) 

I tried to deploy and got the following error::

```
$ make all-in-one-submariner
make: *** No rule to make target 'kind-cni-and-workload', needed by 'all-in-one-submariner'.  Stop.

```

**Comment by dushyantbehl** [comment]
@eranra try now it should work I forgot messaging that it was under development 
Still haven't tested with loadbalancers patch you added and observability yet.

Submariner now works with calico and able to connect as well. Testing needs to be done.

**Comment by nyechiel** [comment]
Hi @dushyantbehl do you have any update on this? Have you managed to give it a try?

**Comment by dushyantbehl** [comment]
Hi @nyechiel was busy with some other threads so will check this next week. Keep you posted. Thanks for bumping me on this :)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+Requirements - 
+1. Go version >= 1.18 Needs to be installed manually
+1. Kubectl -> Needed to be installed manually
+1. jq -> Needs to be installed manually
+1. Kind -> Is setup by the script
+1. Skupper -> Is setup by the script
+1. Submariner -> Is setup by the script
 
+use `make all-in-one-skupper` to deploy skupper environment  
+use `make all-in-one-mbg` to deploy mbg environment
```
Comment (KalmanMeth) [inline README.md:12]: make all-in-one-submariner

**Comment by dushyantbehl** [comment]
@nyechiel with your and Mike's help the debug is done 🥂 
This PR is ready and we can discuss the setup next week or possibly whenever we can schedule a call 😄  after merging this.

**Pull request merged**